src/ape_test/_watch.py

```diff
@@ -79,7 +79,7 @@ def process_event(self, event: events.FileSystemEvent) -> None:
 
 
 def _run_ape_test(*pytest_args):
-    return run_subprocess(["ape", "test", *[f"{a}" for a in pytest_args]])
+    return run_subprocess(["ape", "test", *[f"{a}" for a in pytest_args]], check=False)
```

Comment:
what is the reason for this change? it seems to cause test failures. is this needed?